core/src/main/java/nu/yona/server/subscriptions/service/UserDTO.java

```diff
@@ -53,6 +51,11 @@ public UserDTO(@JsonProperty("firstName") String firstName, @JsonProperty("lastN
 		this(null, firstName, lastName, emailAddress, mobileNumber, privateData);
 	}
 
+	private UserDTO(UUID id, String firstName, String lastName, String mobileNumber, UserPrivateDTO privateData)
+	{
+		this(id, firstName, lastName, null, mobileNumber, privateData);
+	}
+
 	private UserDTO(UUID id, String firstName, String lastName, String emailAddress, String mobileNumber,
```

Comment:
I would like less constructors. Maybe remove email from the constructors and add a setter for this purpose?